ntbs-service/wwwroot/css/header.scss

```diff
@@ -0,0 +1,49 @@
+@import "./colors.scss";
+
+.header-title-and-logo-background {
+    background: white;
+}
+
+.header-navigation-background {
+    background: $phe-red;
+}
+
+.header-navigation-link {
+    padding-right: 40px;
+}
+
+.header-navgation-list {
+    margin-bottom: 0;
+}
+
+@media (min-width: 48.0625em) {
+    .js-enabled .govuk-header__navigation {
+        display: flex;
+        justify-content: space-around;
+    }
+}
+
+.header-title {
+    padding-bottom: 10px;
+    color: $phe-red !important;

```

Comment:
Is this `!important` here becuase we're overriding a property that's already set with `!important`? If not, we should probably just increase the specificity of this to match the rule that's in breaking it for us.
`!importnat` gets out of head really quickly!